shukla/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:22.04
+
+COPY helloWorld.java .
+
+RUN apt-get update && \
+	DEBIAN_FRONTEND=non-interactive \
+	apt-get -y install default-jre-headless && \
+	apt-get clean && \
+	rm -rf /var/lib/apt/lists/* && \
+	apt-get update && apt-get install make 
+
+CMD ["make", "build"] && ["make", "run"] && ["make", "clean"]
```

Comment:
Compiling is an expensive operation which you don't want to have to do at runtime. It would be better to compile the program while building the image and then only run it at container runtime.

Reply:
@urmishukla This suggestion still need to be addressed.